lib/manageiq/providers/ovirt/redhat_rhevm_probe.rb

```diff
@@ -0,0 +1,16 @@
+require 'manageiq/network_discovery/port_scanner'
+
+module ManageIQ::Providers::Ovirt
+  class RedHatRhevmProbe
+    def self.probe(ost)
+      require 'ovirt'
+      Ovirt.logger = $rhevm_log if $rhevm_log
+
+      if PortScanner.portOpen(ost, Ovirt::Service::DEFAULT_PORT)
+        if Ovirt::Service.ovirt?(:server => ost.ipaddr, :verify_ssl => false)
+          ost.hypervisor << :rhevm
```

Comment:
We may want to update to `:ovirt` in another PR.

Reply:
Sorry I'm not following you here.